webapp/pages/profile/_id/_slug.vue

```diff
@@ -64,20 +66,21 @@
               </client-only>
             </ds-flex-item>
           </ds-flex>
-          <ds-space margin="small">
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-28T23:14:04Z" title="Wednesday, January 29th 2020, 12:14:04 am +01:00">Jan 29, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -68,16 +68,23 @@
           </ds-flex>
           <ds-space margin="small">
             <template v-if="!myProfile">
+              <base-button
+                v-if="user.isBlocked"
+                @click="unblockUser(user)"
+                class="unblock-user-button"
+              >
+                {{ $t('settings.blocked-users.unblock') }}
+              </base-button>
+              <base-button v-if="user.isMuted" @click="unmuteUser(user)" class="unmute-user-button">
+                {{ $t('settings.muted-users.unmute') }}
+              </base-button>
```


It's possible that you could see both unblock and unmute buttons, right? I wonder how that would look like :thinking:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-29T09:06:39Z" title="Wednesday, January 29th 2020, 10:06:39 am +01:00">Jan 29, 2020</time>_

---


I uploaded a screen shot, but somehow I added it as a review, and cancelled it, without knowing it would remove it.

![Screenshot from 2020-01-23 19-23-50](https://user-images.githubusercontent.com/26943915/73342473-edbb2980-427e-11ea-90e0-cd21562f6134.png)